web/modules/custom/bnf/bnf_client/src/Entity/Subscription.php

```diff
@@ -56,13 +62,67 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type): a
       ->setRequired(TRUE);
 
     // Using an 'uuid' would seem obvious, but the UUID field generates a new
-    // UUID as default value, so to avoid actidentially generating an UUID, we
+    // UUID as default value, so to avoid accidentally generating an UUID, we
     // use 'string' instead.
     $fields['subscription_uuid'] = BaseFieldDefinition::create('string')
       ->setLabel('Subscription UUID')
       ->setDescription('The UUID subscribed to.')
       ->setRequired(TRUE);
 
+    $fields['tags'] = BaseFieldDefinition::create('entity_reference')
```

Comment:
You're still missing some code to update Drupals definitions. Try running:

```
task dev:bnfcli -- drush php-eval 'print_r(\Drupal::service(\"entity.definition_update_manager\")->getChangeList());'
```

Reply:
Uh.. what? I don't follow. 

```
task: [dev:bnfcli] docker compose -f docker-compose.yml -f docker-compose.bnf.yml run --rm bnfcli sh -c "drush php-eval 'print_r(\Drupal::service(\"entity.definition_update_manager\")->getChangeList());'"
configured lagoon-php.ini
Array
(
    [eventinstance] => Array
        (
            [field_storage_definitions] => Array
                (
                    [title] => 3
                )

        )

)

Reply:
but, maybe this is a non-issue? I guess once we merge it, all of this subscription stuff will be added only on new sites?

Reply:
It worked here, I don't have `last` on the `subscriptions2` branch. But I do have `bnf_source_name`. Have you run updb recently?

It should just be: https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/21738031007ae499a26210573b99c7b316e70ce0/web/modules/custom/bnf/bnf_client/bnf_client.install#L19

Reply:
Yep, `updb` runs as part of `drush deploy` - i even saw it run.
isnt it because you've run ` task dev:reset` whilst on the branch? cause then it creates the site from scratch

Reply:
@xendk I've found a solution here:

https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/2207/commits/2e01b4bf23e9d10d078a84233080004dc3d12a11

I don't know if thats what you had in mind when we talked, but, atleast it works now.

Reply:
That was exactly what I had in mind. I'm sure that someone will point out that it's not correct if the field changes again in the future and all update hooks haven't been run, just as in the old days with `hook_schema` and update hooks, but this is good enough for jazz.